lib/compress.js

```diff
@@ -4652,18 +4652,20 @@ merge(Compressor.prototype, {
             });
         }
         // x ? y(a) : y(b) --> y(x ? a : b)
+        var arg_index;
         if (consequent instanceof AST_Call
             && alternative.TYPE === consequent.TYPE
-            && consequent.args.length == 1
-            && alternative.args.length == 1
             && consequent.expression.equivalent_to(alternative.expression)
```

Comment:
I'm not sure extending this transform to multiple arguments is worth the greatly increased CPU cost when the consequent and alternative are both calls - which is not uncommon. The odds of encountering a successful inversion candidate are already very low and the `equivalent_to` operation is quite expensive relative to bailing out early with a couple of integer comparisons above.

Reply:
https://github.com/mishoo/UglifyJS2/pull/2562#discussion_r155543880 should address this.